src/EPR.RegulatorService.Facade.API/Controllers/OrganisationsSearchController.cs

```diff
@@ -95,12 +105,14 @@ public async Task<IActionResult> OrganisationDetails(Guid externalId)
                 return Ok(organisationDetails);
             }
 
-            _logger.LogError("Fetching organisation details for {externalId} resulted in unsuccessful request: {statusCode}", externalId, response.StatusCode);
+            _logger.LogError("Fetching organisation details for {ExternalId} resulted in unsuccessful request: {StatusCode}", externalId, response.StatusCode);
+
             return HandleError.HandleErrorWithStatusCode(response.StatusCode);
         }
         catch (Exception e)
         {
-            _logger.LogError(e,"Error fetching organisation details for {externalId}", externalId);
+            string logData = string.Format("Error fetching organisation details for {0}", externalId).Replace('\n', '_');
+            _logger.LogError(e, "{Message}", logData);
```

Comment:
delete logData
_logger.LogError(e, "Error fetching organisation details for {0}", externalId);